index.html

```diff
@@ -112,6 +112,7 @@ <h3>Postdocs</h3> <!-- Ordered by last name, Research Scientists and Post Doctor
 <ul>
 <li><a href="https://sites.google.com/site/monsieurlelanc">Subhadeep Banik</a></li>
 <li>Marcel Busch</li>
+<li>Vero Estrada (Senior Research Scientist - DEDIS)</li>
```

Comment:
As you can see with all other names, we don't add lab names to people's names (except for faculty).

Second, we collapse all non-post docs (i.e., those not on a faculty career trajectory) under "software engineers" below. Happy to discuss a name change for the group if necessary.

Reply:
Yes, I think a different name would be appropriate here - better yet a different heading, because Vero doesn't fit under Software engineers nor PostDocs.